### PR TITLE
Initial version without the wanted partitioning

### PR DESCRIPTION
**Pull request by varkoly**

### CODE REVIEW

File: control/installation.xml [hunk 1]
```diff
@@ -7,4 +7,49 @@ textdomain="control"
...
+    </texts>
+    
+    <update>
+      <system_roles>
+        <insert_system_roles config:type="list">
+          <insert_system_role>
+            <system_roles config:type="list">
+              <system_role>
+                  <id>sles4sap_role</id>
+                  <order config:type="integer">0100</order>
```
Comment (jreidinger) [inline control/installation.xml:30]: you can use simple `100`

File: control/installation.xml [hunk 2]
```diff
@@ -7,4 +7,49 @@ textdomain="control"
...
+                  <partitioning>
+                     <proposal_lvm config:type="boolean">true</proposal_lvm>
+                     <try_separate_home config:type="boolean">false</try_separate_home>
+                  </partitioning>
+                  <remove_modules config:type="list">
+                     <remove_module>user_first</remove_module>
+                  </remove_modules>
```
Comment (jreidinger) [inline control/installation.xml:37]: this won't work

File: control/installation.xml [hunk 3]
```diff
@@ -7,4 +7,49 @@ textdomain="control"
...
+                  <insert_modules config:type="list">
```
Comment (jreidinger) [inline control/installation.xml:38]: and same here. Roles is not `<update>` like in addons.

Comment (jreidinger) [reply control/installation.xml:38]: btw travis failed exactly due to this as xml schema does not allow it.

**Comment by teclator** [comment]
@varkoly no updates since @jreidinger reviewed it. Closing by it by now but feel free to reopen it if needed.

**Pull request closed without merging**